securedrop_client/db.py

```diff
@@ -123,6 +129,13 @@ class Message(Base):
                                           cascade="delete"),
                           lazy="joined")
 
+    last_updated = Column(
+        DateTime,
+        nullable=False,
+        default=datetime.datetime.utcnow,
+        onupdate=datetime.datetime.utcnow,
```

Comment:
This introduces a new concept of last_updated for Files, Messages, and Replies, which we also have for Sources. For Sources it represents when a Source was last updated on the server. For Files, Messages, and Replies, it represents when the local database object was last updated. A little documentation about this might go a long way.